src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/SymbolIconTests/SymbolIcon_Foreground_Brushes.xaml

```diff
@@ -0,0 +1,67 @@
+﻿<Page
+    x:Class="UITests.Windows_UI_Xaml_Controls.SymbolIconTests.SymbolIcon_Foreground_Brushes"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:UITests.Windows_UI_Xaml_Controls.SymbolIconTests"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+	<Page.Resources>
+		<LinearGradientBrush x:Key="BrandColor" StartPoint="0,0"
+                           EndPoint="1,1">
+			<GradientStop Color="#fbc64a"
+                       Offset="0.1" />
+			<GradientStop Color="#fbb44b"
+                       Offset="0.2" />
+			<GradientStop Color="#fb8e4d"
+                       Offset="0.4" />
+			<GradientStop Color="#f65745"
+                       Offset="0.5" />
+			<GradientStop Color="#f33140"
+                       Offset="0.6" />
+			<GradientStop Color="#ed2147"
+                       Offset="0.7" />
+			<GradientStop Color="#f2243e"
+                       Offset="0.8" />
+			<GradientStop Color="#ce257f"
+                       Offset="0.9" />
+			<GradientStop Color="#b729a8"
+                       Offset="1.0" />
+		</LinearGradientBrush>
+		<LinearGradientBrush x:Key="BrandColor2" StartPoint="0,0"
+                           EndPoint="1,1">
+			<GradientStop Color="Red"
+                       Offset="0.1" />
+			<GradientStop Color="Green"
+                       Offset="0.2" />
+			<GradientStop Color="Yellow"
+                       Offset="0.4" />
+			<GradientStop Color="Blue"
+                       Offset="0.5" />
+			<GradientStop Color="Black"
+                       Offset="1.0" />
+		</LinearGradientBrush>
+		<SolidColorBrush x:Key="SolidRed" Color="Red" />
+	</Page.Resources>
+	<StackPanel>
+		<SymbolIcon Symbol="Emoji" Foreground="{StaticResource BrandColor}" />
+		<TextBlock x:Name="test1"  Text="Test Test" Foreground="{StaticResource BrandColor}" />
+		<TextBlock x:Name="test2" Text="Test Test Test Test Test Test" Foreground="{StaticResource BrandColor}" />
+
+		<Border BorderBrush="Red" BorderThickness="2">
+			<TextBlock x:Name="test3"  Text="Test Test" Foreground="{StaticResource BrandColor}" />
+		</Border>
+
+		<Border BorderBrush="Red" BorderThickness="2">
+			<TextBlock x:Name="test4" Text="Test Test Test Test Test Test" Foreground="{StaticResource BrandColor}" />
+		</Border>
+
+		<TextBlock>
+            <Run Text="Test Test" Foreground="{StaticResource BrandColor}"/>
+            <Run Text="Test Test Test Test" Foreground="{StaticResource BrandColor2}"/>
+            <Run Text="Test Test" Foreground="{StaticResource SolidRed}"/>
+		</TextBlock>
+	</StackPanel>
```

Comment:
I'm not aware of how to best write a UI test for this. Is there a good way to test gradients? If not, is this a kind of test suitable for marking as manual test?

Reply:
Testing for gradients may be done through large characters and aggressive stops (so that start and finish are of known non shaded colors. We can also use the screenshot tests from the CI.

Reply:
Very nice !